src/Tokenizer/Tokens.php

```diff
@@ -858,6 +860,16 @@ public function isChanged()
         return false;
     }
 
+    public function isEmptyAt($index)
```

Comment:
actually, it's a left-over from a idea that was not decided.
I'm wondering about dropping (on 2.x line deprecating) `Token::isEmpty` in favour of `Tokens:isEmptyAt`.
WDYT ?